TYPO3.Neos/Resources/Private/Partials/Module/Shared/DocumentBreadcrumb.html

```diff
@@ -1 +1,11 @@
-{namespace neos=TYPO3\Neos\ViewHelpers}<neos:backend.documentBreadcrumbPath node="{node}"><f:for each="{documentNodes}" as="documentNode" iteration="{iterator}">/<f:if condition="{showIcon}"><f:if condition="{documentNode.nodeType.ui.icon}"> <i class="{documentNode.nodeType.ui.icon}" title="{f:if(condition: documentNode.nodeType.label, then: '{neos:backend.translate(id: documentNode.nodeType.label)}', else: documentNode.nodeType.name)}" data-neos-toggle="tooltip"></i></f:if></f:if> {documentNode.label} </f:for></neos:backend.documentBreadcrumbPath>
\ No newline at end of file
+{namespace neos=TYPO3\Neos\ViewHelpers}
+<neos:backend.documentBreadcrumbPath node="{node}">
+    <f:for each="{documentNodes}" as="documentNode" iteration="{iterator}">/
+        <f:if condition="{showIcon}">
+            <f:if condition="{documentNode.nodeType.ui.icon}"><i class="{documentNode.nodeType.ui.icon}"
+                                                                 title="{f:if(condition: documentNode.nodeType.label, then: '{neos:backend.translate(id: documentNode.nodeType.label)}', else: documentNode.nodeType.name)}"
+                                                                 data-neos-toggle="tooltip"></i></f:if>
+        </f:if>
+        {documentNode.label}
+    </f:for>
+</neos:backend.documentBreadcrumbPath>
```

Comment:
this adds undesired whitespace between the labels, which is why it was formatted without new lines